<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_find_data">
       <attribute name="icon">
        <iconset theme="magnifying-glass"/>
       </attribute>
       <attribute name="title">
        <string>Find Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="WidgetFindData" name="panel_find_data" native="true">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_my_data">
       <attribute name="icon">
        <iconset theme="street-view"/>
       </attribute>
       <attribute name="title">
        <string>My Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="WidgetMyData" name="panel_my_data" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_upload">
       <attribute name="icon">
        <iconset theme="upload"/>
       </attribute>
       <attribute name="title">
        <string>Uploads</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="UploadWidget" name="panel_upload" native="true">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_download">
       <attribute name="icon">
        <iconset theme="download"/>
       </attribute>
       <attribute name="title">
        <string>Downloads</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="DownloadWidget" name="panel_download" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_maintain">
       <attribute name="icon">
        <iconset theme="broom"/>
       </attribute>
       <attribute name="title">
        <string>Maintenance</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="MaintenanceWidget" name="panel_maintenance" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_logs">
       <attribute name="icon">
        <iconset theme="scroll"/>
       </attribute>
       <attribute name="title">
        <string>Logs</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="WidgetLog" name="panel_logs" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSoftware"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSetupWizard"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuPreferences"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSoftware">
   <property name="icon">
    <iconset theme="code-branch">
     <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Software</string>
   </property>
  </action>
  <action name="actionSet_API_key">
   <property name="text">
    <string>Set DCOR API token</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="times-circle">
     <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUpdate_local_database">
   <property name="text">
    <string>Update local database</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset theme="cogs">
     <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="info">
     <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSetupWizard">
   <property name="icon">
    <iconset theme="hat-wizard">
     <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Setup wizard...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UploadWidget</class>
   <extends>QWidget</extends>
   <header>dcoraid.gui.panel_uploads.widget_upload</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MaintenanceWidget</class>
   <extends>QWidget</extends>
   <header>dcoraid.gui.panel_maintenance.widget_maintenance</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DownloadWidget</class>
   <extends>QWidget</extends>
   <header>dcoraid.gui.panel_downloads.widget_download</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetFindData</class>
   <extends>QWidget</extends>
   <header>dcoraid.gui.panel_find_data.widget_find_data</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLog</class>
   <extends>QWidget</extends>
   <header>dcoraid.gui.logs.widget_log</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetMyData</class>
   <extends>QWidget</extends>
   <header>dcoraid.gui.panel_my_data.widget_my_data</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
